client/merchant-feedback-prompt/negative-modal.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useRef } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Modal } from '@wordpress/components';
+import { dispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { recordEvent } from 'wcpay/tracks';
+import './style.scss';
+
+interface NegativeFeedbackModalProps {
+	onRequestClose: () => void;
+}
+
+export const NegativeFeedbackModal: React.FC< NegativeFeedbackModalProps > = ( {
+	onRequestClose,
+} ) => {
+	const textareaRef = useRef< HTMLTextAreaElement >( null );
+	// Record tracks event when the modal is opened.
+	useEffect( () => {
+		recordEvent( 'wcpay_merchant_feedback_prompt_negative_modal_view' );
+	}, [] );
+
+	return (
+		<Modal
+			title={ __( 'Share your feedback', 'woocommerce-payments' ) }
+			className="wcpay-merchant-feedback-negative-modal"
+			isDismissible={ true }
+			shouldCloseOnClickOutside={ false } // Should be false because of the iframe.
+			shouldCloseOnEsc={ true }
+			onRequestClose={ () => {
+				recordEvent(
+					'wcpay_merchant_feedback_prompt_negative_modal_close_click'
+				);
+				onRequestClose();
+			} }
+		>
+			<div className="wcpay-merchant-feedback-negative-modal__content">
+				<p>
+					{ __(
+						'Thanks for sharing your feedback on WooPayments! Your feedback helps us to continue to improve and deliver the best tools for your business.',
+						'woocommerce-payments'
+					) }
+				</p>
+				<p className="wcpay-merchant-feedback-negative-modal__question">
+					{ __(
+						'Would you mind sharing more about why you chose that option?',
+						'woocommerce-payments'
+					) }
+				</p>
+				<textarea
+					ref={ textareaRef }
+					className="wcpay-merchant-feedback-negative-modal__textarea"
+					placeholder={ __(
+						'Share your feedback here…',
+						'woocommerce-payments'
+					) }
+				/>
+				<p className="wcpay-merchant-feedback-negative-modal__privacy">
+					{ __(
+						'Your feedback will be sent to the WooCommerce team. Your personal information is secure and will not be shared with third parties. For more details, please see our ',
```

Comment:
I've made a small tweak here to add the missing ' ' after 'please see our' in 399d9aa04e1f9cd376ffd2ed003d6a937e3ebfd3

Reply:
Good catch! Thank you.